src/pages/Cart/Context/CartContext.jsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable react/prop-types */
```

Comment:
@AYOPELUMI 
Here you disabled the eslint from the project. please kindly do your implementation without disabling the eslint. the eslint helps us write good codes that are modular and obeys standards.

Reply:
Thanks for the insights 
I will do that after fully pleased that all basic functionality and design layout are met